WordPress/Classes/ViewRelated/Blog/Blog Dashboard/BlogDashboardViewController.swift

```diff
@@ -325,20 +325,6 @@ extension BlogDashboardViewController {
     }
 }
 
-// MARK: - UI Popover Delegate
-
-/// This view controller may host a `DashboardPromptsCardCell` that requires presenting a `MenuSheetViewController`,
-/// a fallback implementation of `UIMenu` for iOS 13. For more details, see the docs on `MenuSheetViewController`.
-///
-/// NOTE: This should be removed once we drop support for iOS 13.
-///
-extension BlogDashboardViewController: UIPopoverPresentationControllerDelegate {
-    // Force popover views to be presented as a popover (instead of being presented as a form sheet on iPhones).
-    public func adaptivePresentationStyle(for controller: UIPresentationController, traitCollection: UITraitCollection) -> UIModalPresentationStyle {
-        return .none
-    }
-}
-
```

Comment:
This is the only change I'm not sure of. It looks like it's the Quick Start popover but I'm not sure how it's relevant to dropping iOS 14. 
Do you have more insight into this, @kean?

Reply:
The only code in Dashboard that I found that made use of this delegate is also removed as part of this PR:

```swift
// Fallback context menu implementation for iOS 13.
func showMenuSheet() {
    // ...
   
    if let popoverPresentationController = menuViewController.popoverPresentationController {
            popoverPresentationController.delegate = presenterViewController

   // ...
}
```

I'm planning to remove `MenuSheetViewController` as well, but there is one use left in `ReaderCommentsViewController`, which I'm not sure is safe to remove – I'll investigate.

Reply:
> You have to set the delegate in order to it to get called, so it seems it's safe to remove.

Gotcha 👍 



> I'm planning to remove [`MenuSheetViewController`](https://github.com/wordpress-mobile/WordPress-iOS/blob/trunk/WordPress/Classes/ViewRelated/Views/MenuSheetViewController.swift) as well, but there is one use left in `ReaderCommentsViewController`, which I'm not sure is safe to remove – I'll investigate.

Thanks for taking a look.

I'll approve now since this PR looks good!